corehq/apps/app_manager/feature_support.py

```diff
@@ -208,3 +208,10 @@ def enable_exclude_from_search(self):
             toggles.USH_CASE_CLAIM_UPDATES.enabled(self.domain)
             and self._require_minimum_version('2.53')
         )
+
+    @property
+    def enable_required_search_fields(self):
```

Comment:
I'm thinking about combining this with `enable_exclude_from_search`, just haven't come up with a good descriptive name yet.

Reply:
"ush_case_claim_updates2" :trollface:

Reply:
😮